Tools/scripts/serve.py

```diff
@@ -28,7 +28,7 @@ def app(environ, respond):
     path = sys.argv[1]
     port = int(sys.argv[2]) if len(sys.argv) > 2 else 8000
     httpd = simple_server.make_server('', port, app)
-    print("Serving {} on port {}, control-C to stop".format(path, port))
+    print("Serving {0} on 0.0.0.0 port {1} (http://0.0.0.0:{1}/), control-C to stop".format(path, port))
```

Comment:
What do you think about using an f-string here?

Reply:
I could use a f-string but you can not build the documentation with python 3.5 (not compliant with the f-string)

Reply:
I see, thanks!